Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -572,7 +572,9 @@ void RenderBlockFlow::layoutBlock(bool relayoutChildren, LayoutUnit pageLogicalH
     LayoutUnit newHeight = logicalHeight();
 
     LayoutUnit alignContentShift = 0_lu;
-    if ((!isPaginated || pageRemaining > newHeight) && (settings().alignContentOnBlocksEnabled()))
+    // alignment isn't supported when fragmenting
+    // table cell alignment is handled in RenderTableCell::computeIntrinsicPadding
```

Comment:
WebKit style guide says start of comments should be capitalized and end should have a period

Reply:
Done!